cmake/DAGMCConfig.cmake.in

```diff
@@ -0,0 +1,19 @@
+# - Config file for the Direct Accelerated Geometry Monte Carlo (DAGMC) package
+# It defines the following variables
+#  DAGMC_INCLUDE_DIRS - include directories for installed DAGMC headers
+#  DAGMC_LIBRARY_DIRS - location of installed DAGMC libraries
+#  DAGMC_LIBRARIES    - set of libraries installed with DAGMC, use to link applications against DAGMC
+
+# Declares the DAGMC package as found by CMake
+set(DAGMC_FOUND 1)
+
+set(DAGMC_MAJOR_VERSION @DAGMC_MAJOR_VERSION@)
+set(DAGMC_MINOR_VERSION @DAGMC_MINOR_VERSION@)
+set(DAGMC_PATCH_VERSION @DAGMC_PATCH_VERSION@)
+set(DAGMC_VERSION @DAGMC_VERSION@)
+ 
+message(STATUS "Configuring DAGMC ${DAGMC_VERSION}")
+
+set(DAGMC_INCLUDE_DIRS "@CMAKE_INSTALL_PREFIX@/@INSTALL_INCLUDE_DIR@" "@MOAB_INCLUDE_DIRS@")
+set(DAGMC_LIBRARY_DIRS "@CMAKE_INSTALL_PREFIX@/@INSTALL_LIB_DIR@" "@MOAB_LIBRARY_DIRS@")
+set(DAGMC_LIBRARIES dagmc dagtally makeWatertight pyne_dagmc @MOAB_LIBS@)
```

Comment:
Any reason you're using MOAB_LIBS instead of MOAB_LIBRARIES? Not sure what the difference is

Reply:
MOAB uses that variable name to create a list of its compiled directories rather than MOAB_LIBRARIES. I could use MOAB_LIBRARIES but I'd have to remove the path definitions from it. This is easier and clearly defined in MOAB's MOABConfig.cmake. 

It also contains `-l` in front of library names, but I presume that CMake is handling that well based on my local tests and hopefully CI too.